systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
```

Comment:
This is a nice initiate from you to set some strings translatable, but Systers Portal is English-only and probably won't change in the near future, so you can safely remove all translation support.